doc/mediasdk-man.md

```diff
@@ -8994,6 +9033,26 @@ The `Protected` enumerator describes the protection schemes.
 **Change History**
 
 This enumerator is available since SDK API 1.30.
+
+## <a id='PartialBitstreamOutput'>PartialBitstreamOutput</a>
+
+**Description**
+
+The `PartialBitstreamOutput` enumerator indicates flags of partial bitstream output type. 
+
+**Name/Description**
+
+| | |
+--- | ---
+`MFX_PARTIAL_BITSTREAM_NONE`  | Don't use partial output
+`MFX_PARTIAL_BITSTREAM_SLICE` | Partial bitstream output will be aligned to slice granularity (AVC) 
+`MFX_PARTIAL_BITSTREAM_BLOCK` | Partial bitstream output will be aligned to user-defined block size granularity
+`MFX_PARTIAL_BITSTREAM_ANY` | Partial bitstream output will be return any coded data avilable at the end of SyncOperation timeout
```

Comment:
It sounds that "MFX_PARTUAL_BITSTREAM_FULL" will be better since it highlights that all data should be returned, i.e. no partial

Reply:
I am not agree. FULL means full bitstream while ANY means any incomplete data available upon sync operation call

Reply:
I misread the sentence. Please, fix typos to make it readable:
* s/will be return/will return/
* s/avilable/available/

> at the end of SyncOperation timeout

I think this requires explicit comment in the man. Do I understand correctly that if I will call SyncOperation(timeout) then on the point of exit from SyncOperation with WRN_IN_EXECUTION I will get some data in the buffer? If yes, do you have example in the manual how to use that? Which timeout to specify?

Reply:
done